Remotion/ObjectBinding/Web/UI/Controls/BocListEvents.cs

```diff
@@ -155,6 +161,13 @@ public void SetAdditionalCssClassForDataRow (string cssClass)
     ArgumentUtility.CheckNotEmpty("cssClass", cssClass);
     _additionalCssClassForDataRow = cssClass;
   }
+
+  public void SetAdditionalCssClassForValidationRow (string cssClass)
+  {
+    ArgumentUtility.CheckNotEmpty("cssClass", cssClass);
+    _additionalCssClassForValidationRow = cssClass;
+  }
+
```

Comment:
Hmm...I don't think we need `AdditionalCssClassForValidationRow`. Instead, the `AdditionalCssClassForDataRow` would also get applied to the validation row. The idea is to have both rows styles identically. If someone really did want special treatment, they can also use `tr.validationRow.myCustomRowCssClass` to slove this.